### ISSUE TITLE
Add integrity rules

### ISSUE THREAD

**Issue by nichtich** [gbv/daia#9]
Entities, document, and items are all identified by URIs.
- If both department and storage have the same id, storage SHOULD be ignored
- If both document and item have the same id ...

**Comment by nichtich** [comment]
This is allowed but discouraged. The spec should tell how to handle it.

```
{
   "available": [
    { "service": "presentation" },
    { "service": "loan" }
  ],
  "unavailable": [
    { "service": "presentation" },
    { "service": "loan" }
  ]
}
```

This, on the other hand, makes sense:

```
{
   "available": [
    { "service": "presentation", "limitation": [ ... ] },
    { "service": "loan", "limitation": [ ... ] }
  ],
  "unavailable": [
    { "service": "presentation" },
    { "service": "loan" }
  ]
}
```

**Comment by nichtich** [comment]
Included in 0.9.3
